obscuro-ethereum-interaction.md

```diff
@@ -60,3 +61,16 @@ The Bridge contract will keep track of these requests and will execute them at d
 
 The withdrawal process is indicated in the following diagram:
 ![withdrawal process](./images/withdrawal-process.png)
+
+
+### Obscuro public events
+
+Applications running inside Obscuro can emit special types of events called _Public Events_, which the OVM will add in plaintext into a dedicated data structure in the rollup.
```

Comment:
The glossary should be updated to include "Public Events". Want to add that in this PR too?

Reply:
yes, good point.